types/relay-compiler/tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "lib": [
-            "dom",
-            "es6",
-            "esnext.asynciterable"
-        ],
+        "lib": ["ES6"],
```

Comment:
I think that if you convert this to `"es6"`, the bot shouldn't complain about the config anymore.